jsinterop-ts-defs-processor/pom.xml

```diff
@@ -138,6 +138,46 @@
                     <target>8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.4.1</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <artifactSet>
+                                <excludes>
+                                   <exclude>com.vertispan.tsdefs:jsinterop-ts-defs</exclude>
+                                    <exclude>com.google.jsinterop:*</exclude>
+                                    <exclude>javax.annotation:*</exclude>
+                                    <exclude>javax.validation:*</exclude>
+                                    <exclude>com.google.code.findbugs:jsr305</exclude>
+                                    <exclude>com.google.j2objc:j2objc-annotations</exclude>
+                                    <exclude>com.google.errorprone:error_prone_annotations</exclude>
+                                </excludes>
+                            </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>com.google</pattern>
+                                    <shadedPattern>jsinterop.ts.def.shaded.com.google</shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>org.checkerframework</pattern>
+                                    <shadedPattern>jsinterop.ts.def.shaded.org.checkerframework</shadedPattern>
```

Comment:
okay with nesting this in com.vertispan.tsdefs instead?

Reply:
Yup, updated